### ISSUE TITLE
raspberry workstations: 

### ISSUE THREAD

**Issue by FranGomezVenegas** [FranGomezVenegas/FrontE-Issues#60]
For certain tasks like temperature reading ... if they want to enter the value manually ... then keep a session open all time with a single screen and then enter the value + user confirmation without platform

In some way the demo for components is the way. The only one thing is that the window should open with no userlogin to be accessible for everyone but then it is when whoever perform the action should enter the credentials, how?by user confirm credential checker, then ee will inform the user credential checker to let the BE to create the token and perform the endpoint if everything is fine. 
In other words, if the regular basis is that the token si generated during the log-in, in this case the token is generated during the action trigger through the user credentials, to get the user.

**Comment by FranGomezVenegas** [comment]
created platform Workstations

**Comment by FranGomezVenegas** [comment]
Pending, display user login view to let the user set the user credentials instead of using a permanent user session
